boot/zephyr/CMakeLists.txt

```diff
@@ -70,10 +70,10 @@ assert_exists(MBEDTLS_ASN1_DIR)
 set(NRF_DIR "${MCUBOOT_DIR}/ext/nrf")
 
 if(CONFIG_BOOT_USE_NRF_CC310_BL)
-set(NRFXLIB_DIR ${MCUBOOT_DIR}/../nrfxlib)
+set(NRFXLIB_DIR ${MCUBOOT_DIR}/../../nrfxlib)
```

Comment:
What about using `${ZEPHYR_BASE}/..` instead?

Reply:
Sure, that is used elsewhere in this file as well. It must be ENV${ZEPHYR in that case as it is not available as a cmake variable just yet.

Reply:
Fixed, the other define was not used anywhere, so I simply removed it

Reply:
Could you split the changes into two commits, and submit the one replacing `${MCUBOOT_DIR}` with `ENV${ZEPHYR}` to upstream as well?

Reply:
fixed